pkg/commands/build/status.go

```diff
@@ -27,15 +26,12 @@ import (
 
 const (
 	buildMetadataKey = "io.buildpacks.build.metadata"
-	bomKey           = "bom"
 )
 
 func NewStatusCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilProvider) *cobra.Command {
```

Comment:
oh wait I missed this before. The `registry.UtilProvider` can be removed as well

Reply:
Thanks! Please see 831e0496cbc9a6ee8da4e5a997a5487d6c0a8714.